include/arch/arm-imx/arch.h

```diff
@@ -33,79 +33,7 @@
 #define __STRNCPY
 #define __MEMMOVE
 
-
-#define max(a, b) ({ \
-  __typeof__ (a) _a = (a); \
-  __typeof__ (b) _b = (b); \
-  _a > _b ? _a : _b; })
-
-#define min(a, b) ({ \
-  __typeof__ (a) _a = (a); \
-  __typeof__ (b) _b = (b); \
-        _a > _b ? _b : _a; \
-})
-
-
-typedef unsigned char u8;
-typedef unsigned short u16;
-typedef unsigned int u32;
-typedef unsigned long long u64;
-
-typedef signed char s8;
-typedef short s16;
-typedef int s32;
-typedef long long s64;
-
-typedef u32 addr_t;
-typedef u64 cycles_t;
-typedef u32 uintptr_t;
-typedef s32 intptr_t;
-
-typedef int ssize_t;
-typedef unsigned long long time_t;
-typedef unsigned int useconds_t;
-
-typedef u64 offs_t;
-
-typedef u64 id_t;
-typedef struct _oid_t {
-	u32 port;
-	id_t id;
-} oid_t;
-
-
-typedef u32 handle_t;
-
-#include <limits.h>
-
-#define SIZE_PAGE 0x1000
-
-#define PRId8       "d"
-#define PRIu8       "u"
-#define PRIx8       "x"
-#define PRIX8       "X"
-#define PRIi8       "i"
-#define PRIo8       "o"
-
-#define PRId16      "d"
-#define PRIu16      "u"
-#define PRIx16      "x"
-#define PRIX16      "X"
-#define PRIi16      "i"
-#define PRIo16      "o"
-
-#define PRId32      "d"
-#define PRIu32      "u"
-#define PRIx32      "x"
-#define PRIX32      "X"
-#define PRIi32      "i"
-#define PRIo32      "o"
-
-#define PRId64      "lld"
-#define PRIu64      "llu"
-#define PRIx64      "llx"
-#define PRIX64      "llX"
-#define PRIi64      "lli"
-#define PRIo64      "llo"
+#define _PAGE_SIZE 0x1000
```

Comment:
Would be great to have this value from a kernel header, if possible.

Reply:
Yeah, I think that it can be achieved along with posix revamp in kernel where many headers will be changed.